assets/js/mbta.ts

```diff
@@ -3135,15 +3169,19 @@ const arincToRealtimeIdMap: { [key: string]: string } = {
   'GSCI-e': 'science_park_eastbound',
   'GSCI-w': 'science_park_westbound',
   'GSCI-m': 'science_park_mezzanine',
-  'GLEC-w': 'lechmere_green_line',
+  'GLEC-e': 'lechmere_green_line_eastbound',
+  'GLEC-w': 'lechmere_green_line_westbound',
+  'GLEC-m': 'lechmere_green_line_mezzanine',
+  'GUNS-e': 'union_sq_north',
+  'GUNS-w': 'union_sq_south',
```

Comment:
Doesn't this have to match the realtime_signs config? So either this should be `east` and `west`, or realtime_signs should be `north` and `south`.

Reply:
oops yes, I had made those but forgot to push them. Thanks for calling that out!